### ISSUE TITLE
Multiple LUA errors

### ISSUE THREAD

**Issue by Skunkzord** [liquidbase/DuffedUIv8#78]
These LUA errors I found when running the new DuffedUI.
I think seeing the code that some of them do not have alot to do with stuff from DuffedUI and I must say my installation is not clean since the patch so I might delete all addons and use a clean installation, but at least you can get an idea what I encountered 👍 

LUA error number 1:

```
Message: ...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: attempt to call upvalue 'RegisterAddonMessagePrefix' (a nil value)
Time: Mon Jul 23 21:41:55 2018
Count: 1
Stack: ...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: attempt to call upvalue 'RegisterAddonMessagePrefix' (a nil value)
...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: in function `RegisterPlugin'
...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:200: in main chunk

Locals: self = <table> {
 GetPluginOptions = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:105
 VersionCheck = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:123
 groupSize = -1
 GenerateVersionCheckMessage = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:109
 prefix = "ElvUIPluginVC"
 GeneratePluginList = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:156
 plugins = <table> {
 }
 index = 0
 RegisterPlugin = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:83
 SendPluginVersionCheck = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:178
}
name = "LibElvUIPlugin-1.0"
callback = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:105
plugin = <table> {
 callback = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:105
 version = 20
 name = "LibElvUIPlugin-1.0"
}
(*temporary) = nil
(*temporary) = "ElvUIPluginVC"
(*temporary) = "attempt to call upvalue 'RegisterAddonMessagePrefix' (a nil value)"
MAJOR = "LibElvUIPlugin-1.0"
MINOR = 20
GetAddOnMetadata = <function> defined =[C]:-1
lib = <table> {
 GetPluginOptions = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:105
 VersionCheck = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:123
 groupSize = -1
 GenerateVersionCheckMessage = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:109
 prefix = "ElvUIPluginVC"
 GeneratePluginList = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:156
 plugins = <table> {
 }
 index = 0
 RegisterPlugin = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:83
 SendPluginVersionCheck = <function> defined @Interface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:178
}
RegisterAddonMessagePrefix = nil
CreateFrame = <function> defined =[C]:-1
Options = <table> {
 type = "group"
 name = "Plugins"
 args = <table> {
 }
 order = -10
 guiInline = false
}
```

LUA error number 2:

```
Message: ...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: attempt to call upvalue 'RegisterAddonMessagePrefix' (a nil value)
Time: Mon Jul 23 21:41:58 2018
Count: 1
Stack: ...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: attempt to call upvalue 'RegisterAddonMessagePrefix' (a nil value)
...erface\AddOns\ProjectAzilroka\LibElvUIPlugin-1.0.lua:90: in function `RegisterPlugin'
Interface\AddOns\ProjectAzilroka\Init.lua:237: in function `?'
...ibs\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:145: in function <...ibs\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:145>
[string "safecall Dispatcher[1]"]:4: in function <[string "safecall Dispatcher[1]"]:4>
[C]: ?
[string "safecall Dispatcher[1]"]:13: in function `?'
...ibs\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: in function `Fire'
...jectAzilroka\Libs\Ace3\AceEvent-3.0\AceEvent-3.0.lua:120: in function <...jectAzilroka\Libs\Ace3\AceEvent-3.0\AceEvent-3.0.lua:119>

Locals: <none>
```

LUA error number 3:

```
Message: Error loading Interface\AddOns\AddOnSkins\Core\Embed.lua
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 4:

```
Message: Error loading Interface\AddOns\AddOnSkins\Skins\Blizzard\Transmogrify.lua
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 5:

```
Message: Error loading Interface\AddOns\AddOnSkins\Skins\Blizzard\Tutorial.lua
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 6:

```
Message: Error loading Interface\AddOns\AddOnSkins\Skins\Blizzard\WorldStateScoreFrame.lua
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 7:

```
Message: Couldn't open Interface\AddOns\AddOnSkins\
Time: Mon Jul 23 21:41:59 2018
Count: 3
```

LUA error number 8:

```
Message: Deferred XML Node object named Lib_UIDropDownMenuButtonTemplate already exists
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 9:

```
Message: Deferred XML Node object named Lib_UIDropDownListTemplate already exists
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 10:

```
Message: Deferred XML Node object named Lib_UIDropDownMenuTemplate already exists
Time: Mon Jul 23 21:41:59 2018
Count: 5
```

LUA error number 11:

```
Message: integer overflow attempting to store -nan(ind)
Time: Mon Jul 23 21:43:01 2018
Count: 7
Stack: integer overflow attempting to store -nan(ind)
[C]: in function `format'
...nterface\AddOns\DuffedUI\modules\misc\experience.lua:127: in function <...nterface\AddOns\DuffedUI\modules\misc\experience.lua:113>

Locals: (*temporary) = "Rep: %s/%s (%d%%)"
(*temporary) = "0"
(*temporary) = "0"
(*temporary) = -nan(ind)
```

**Comment by Potsrabe** [comment]
Hello Skunkzork,
For first time You have to deaktivate ProjectAzilroka and AddOnSkins. Because this moduls are not yet remastered for BfA. This wil liquid do when all other works.

BR Ralf

**Comment by liquidbase** [comment]
Issues from AddOnSkins and ProjektAzilroka should be fixed. If not please report back. Issue with reputationbar on the UI is unfixed. Possible tomorrow.

**Comment by Potsrabe** [comment]
Hello,
I have this Errors:
1.) 1x DuffedUI\modules\skins\greeting.lua:15: function arguments expected near ':'
Locals:
2.) 1x DuffedUI\core\cversion.lua:50: attempt to call local 'C_ChatInfo_RegisterAddonMessagePrefix' (a nil value)
DuffedUI\core\cversion.lua:50: in main chunk

And this after load:
ProjectAzilroka: Frame existiert nicht: WorldStateAlwaysUpFrame
AddOnSkins Version: 3.74 Loaded!
AddOnSkins 3.74: There was an error in the Blizzard_Options skin.
AddOnSkins 3.74: There was an error in the Blizzard_PvE skin.
AddOnSkins 3.74: There was an error in the Blizzard_Quest skin.
AddOnSkins 3.74: There was an error in the Blizzard_WorldMap skin.

In the Version I have download yesterday without AddOnSkins and ProjectAzilroka no Errors.

And with ProjectAzilroka I have the following Problem. Square MinimapButtons can be activate and do working, but I can not move it. 

An than I have see today. On Info Center no Time found to open the Calendar quickly.

BR Ralf

**Comment by Skunkzord** [comment]
First of all, thanks on the fast update, i really do appreciate it and I think alot of other people do too. :)

Awesome Liquidbase, it works now. I did a fresh installation deleted all addons etc and it seems now most issues are solved from addonskins and azilroka.  Some textures are still being displayed like the blizzard ones though, I don't know why, but I guess those addons need to be updated as well.

Also I discovered a peculiar issue with the Chat frame on the left side of the screen, while scrolling with the mouse button through the main frame I see the two blizzard style buttons in the lower right corner of the screen. But this is a minor issue, since they dissapear after the fade out kicks in.
